### ISSUE TITLE
Provide option to create JReleaser configuration

### ISSUE THREAD

**Issue by gunnarmorling** [moditect/oss-bootstrap#9]
I'm not sure whether we always should create that by default, but it may be an opt-in feature which can be chosen when creating a project from the archetype. WDYT, @aalmiray?

**Comment by aalmiray** [comment]
Well, I think archetypes cannot modify the contents of the `pom.xml` _unless_ we add a post processing Groovy script. Are you up to that? 😼

**Comment by gunnarmorling** [comment]
Yes, I think there will be no way around that. A few of the customizations will require it in all likelihood, e.g. adjusting license headers in the generated files as per the selected license (see #2).

**Comment by gunnarmorling** [comment]
Hey @aalmiray, so with the basic archetype in place, do you think you could give it a shot to update the template to include all the required configuration for releasing to Maven Central? I can take care of archetype-izing it then, if needed. WDYT?

**Comment by aalmiray** [comment]
OK, I'll see what I can do.
